src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -79,6 +79,12 @@
 
   private static final String DEFAULT_GAPIC_NAME = "gapic";
   private static final String DEFAULT_GAPIC_VERSION = "";
+  private static final String ARTIFACT_ID = "{@xsettingsClass.artifactId}";
+  private static final String GROUP_ID = "com.google.cloud";
+  private static final String META_FILE_ROOT = "/META-INF/maven/";
```

Comment:
At a more general level, I'm not comfortable with assuming that our generated code will be compiled with maven.